### PR TITLE
(improvement) Remove historical order selection on clickout

### PR DESCRIPTION
**Pull request by Blaumaus**
### Asana task:
[[GHI-786] Remove historical order selection on clickout](https://app.asana.com/0/1201849173362898/1204061504712510/f)

### UI Demonstration:
https://user-images.githubusercontent.com/35810911/221932683-6c62e82f-6584-4be2-8b97-553be45e51f5.mov

### CODE REVIEW

**Review by dmytroshch — CHANGES_REQUESTED** [review]
Due to specs `remove the number of AOs from the title if it's history, this way the user know the number always informs about active orders`

File: src/components/TradingStatePanel/style.scss [hunk 1]
```diff
@@ -1,6 +1,10 @@
 @import "../../variables.scss";
 
 .hfui-tradingstatepanel__wrapper {
+  .hfui-panel__inner > div {
+    height: 100%;
+  }
+
```
Comment (dmytroshch) [inline src/components/TradingStatePanel/style.scss:7]: Can we do selectors more specific, using classes?

File: src/components/AlgoOrdersTable/style.scss [hunk 2]
```diff
@@ -4,6 +4,10 @@
   padding-bottom: 8px;
   height: 100%;
 
+  > div {
+    height: 100%;
+  }
+
```
Comment (dmytroshch) [inline src/components/AlgoOrdersTable/style.scss:10]: Can we do selectors more specific, using classes?

File: src/components/TradingStatePanel/TradingStatePanel.container.js [hunk 3]
```diff
@@ -4,15 +4,17 @@ import { reduxSelectors } from '@ufx-ui/bfx-containers'
 import {
   getFilteredPositionsCount,
   getFilteredAtomicOrdersCount,
-  getFilteredAlgoOrdersCount,
```
Comment (dmytroshch) [inline src/components/TradingStatePanel/TradingStatePanel.container.js:7]: seems this selector `get_filtered_algo_orders_count.js` isn't used elsewhere in code and should be removed

**Review by dmytroshch — CHANGES_REQUESTED** [review]

File: src/components/TradingStatePanel/style.scss (hunk 1 above)
Comment (Blaumaus) [reply src/components/TradingStatePanel/style.scss]: This div is generated by `OutsideClickHandler` component, as far as I know it does not support className props

**Comment by bhoomij** [comment]
@Blaumaus please help fix conflicting files

**Review by bhoomij — APPROVED** [review]

**Pull request merged**